README.md

```diff
@@ -182,13 +182,18 @@ Pyro users will note that the API for model specification and inference is large
 
 ## Installation
 
-> **Limited Windows Support:** Note that NumPyro is untested on Windows, and will require building jaxlib from source. See this [JAX issue](https://github.com/google/jax/issues/438) for more details.
+> **Limited Windows Support:** Note that NumPyro is untested on Windows, and might require building jaxlib from source. See this [JAX issue](https://github.com/google/jax/issues/438) for more details. Alternatively, you can install [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/) as use NumPyro on it as on a Linux system. See also [CUDA on Windows Subsystem for Linux](https://developer.nvidia.com/cuda/wsl) if you want to use GPUs on Windows.
 
 To install NumPyro with a CPU version of JAX, you can use pip:
 
 ```
 pip install numpyro
 ```
+or
+```
+pip install numpyro[funsor]
+```
+if you want to do inference for models with discrete latent variables.
```

Comment:
I have changed README a bit to mention about Windows limitedness and funsor installation.

Reply:
I think this installation method (`pip install numpyro[funsor]`) should be in `installing from source` below?

Reply:
yes, will change